flask_openapi3/openapi.py

```diff
@@ -204,6 +208,9 @@ def api_doc(self) -> Dict:
             The OpenAPI specification JSON as a dictionary.
 
         """
+        if self.spec_json:
```

Comment:
I see, this is how you would change the default schema to not have the 422 then. I understand.